### PR TITLE
Add some basic instructions for building the BasicMinimalApi project locally

### PR DESCRIPTION
**Pull request by amcasey**
Just recording @eerhardt's wisdom for posterity. 😄

### CODE REVIEW

**Review by sebastienros — APPROVED** [review]

File: src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+In order to avoid interfering with your other uses of dotnet, it's probably preferable to set those variables only in the prompt where you're building this project.
+
+Confirm that things are working by running `dotnet --version` - the output should match the build that you downloaded.
+Tools like `which` and `where` can also help confirm that you're running the version you expect.
+
+## Building
+
+In a prompt with `PATH` and `DOTNET_ROOT` set appropriately, run the following ocmmand
```
Comment (eerhardt) [inline src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md:26]:
```suggestion
In a prompt with `PATH` and `DOTNET_ROOT` set appropriately, run the following command
```

File: src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+dotnet publish /p:PublishAot=true /p:StripSymbols=true /p:EnableRequestDelegateGenerator=true
+```
+The properties and their values come from [goldilocks.benchmarks.yml](../../../scenarios/goldilocks.benchmarks.yml).
+
+The build should complete without warnings or errors.
```
Comment (eerhardt) [inline src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md:32]:
```suggestion
The build should complete without errors.
```

Today there still are a few warnings. :(

File: src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Confirm that things are working by running `dotnet --version` - the output should match the build that you downloaded.
+Tools like `which` and `where` can also help confirm that you're running the version you expect.
+
+## Building
+
+In a prompt with `PATH` and `DOTNET_ROOT` set appropriately, run the following ocmmand
+```
+dotnet publish /p:PublishAot=true /p:StripSymbols=true /p:EnableRequestDelegateGenerator=true
+```
+The properties and their values come from [goldilocks.benchmarks.yml](../../../scenarios/goldilocks.benchmarks.yml).
```
Comment (eerhardt) [inline src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md:30]:
```suggestion
The properties and their values come from the `basicminimalapipublishaot` scenario in [goldilocks.benchmarks.yml](../../../scenarios/goldilocks.benchmarks.yml).
```

**Review by eerhardt — APPROVED** [review]
Thanks!

File: src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md (hunk 2 above)
Comment (amcasey) [reply src/BenchmarksApps/BasicMinimalApi/BuildingLocally.md:32]: 😢

**Pull request merged**